Source/WebCore/Headers.cmake

```diff
@@ -1534,7 +1534,7 @@ set(WebCore_PRIVATE_FRAMEWORK_HEADERS
     page/RemoteFrame.h
     page/RemoteFrameClient.h
     page/RemoteFrameView.h
-    page/RemoteMouseEventData.h
+    page/RemoteInputEventData.h
```

Comment:
Nit - `InputEvent` actually refers to a different type of editing-related DOM event in the specification, so this name might be a bit confusing; since `WebCore::DragEvent` extends from `WebCore::MouseEvent`, it might be okay to just keep using `RemoteMouseEventData` for drag events; alternately, you could name it something like `RemoteUserInputEventData` or `RemoteMouseInputEventData`

Reply:
This same struct will probably also be used for touch events in the future. I'll rename to be `RemoteUserInputEventData`.